### PR TITLE
fix softmax error

### PR DESCRIPTION
**Pull request by weilinquan8**

### CODE REVIEW

File: examples/BuddyLlama/CMakeLists.txt [hunk 1]
```diff
@@ -63,14 +63,4 @@ SET_TARGET_PROPERTIES(
...
-  LLAMA
-  mlir_c_runner_utils
-  omp
-)
-if(BUDDY_MLIR_USE_MIMALLOC)
-  list(APPEND BUDDY_LLAMA_LIBS mimalloc)
-endif()
-
-target_link_libraries(buddy-llama-run ${BUDDY_LLAMA_LIBS})
+target_link_libraries(buddy-llama-run LLAMA)
```
Comment (zhanghb97) [inline examples/BuddyLlama/CMakeLists.txt:66]: The `LLAMA` target linked libraries retain the previous version. (use `BUDDY_LLAMA_LIBS`)

File: examples/BuddyLlama/llama-main.cpp [hunk 2]
```diff
@@ -20,13 +20,15 @@
...
+#include <limits>
+#include <type_traits>
 
 using namespace buddy;
 using namespace std;
 using namespace chrono;
 
 extern "C" void _mlir_ciface_forward(MemRef<float, 3> *, MemRef<float, 1> *,
-                                     MemRef<size_t, 2> *);
+                                     MemRef<long long, 2> *);
```
Comment (zhanghb97) [inline examples/BuddyLlama/llama-main.cpp:31]: I think `size_t` is the more suitable option here. What is the advantage of using `long long`?

Comment (weilinquan8) [reply examples/BuddyLlama/llama-main.cpp:31]: Yes, I will update this.

File: examples/BuddyLlama/CMakeLists.txt (hunk 1 above)
Comment (weilinquan8) [reply examples/BuddyLlama/CMakeLists.txt:66]: OK

File: tests/Python/test_t.py [hunk 3]
```diff
@@ -25,8 +25,13 @@ def foo(x):
 
 # CHECK: module {
 # CHECK-LABEL: func.func @forward
+<<<<<<< HEAD
```
Comment (zhanghb97) [inline tests/Python/test_t.py:28]: Wrong code. Please remove these. Same thing in the following file.

File: examples/BuddyLlama/CMakeLists.txt [hunk 4]
```diff
@@ -48,29 +47,29 @@ add_custom_command(
...
-  mlir_c_runner_utils
-  omp
-)
-if(BUDDY_MLIR_USE_MIMALLOC)
-  list(APPEND BUDDY_LLAMA_LIBS mimalloc)
-endif()
+  add_library(LLAMA STATIC llama.o)
 
-target_link_libraries(buddy-llama-run ${BUDDY_LLAMA_LIBS})
+  SET_SOURCE_FILES_PROPERTIES(
```
Comment (zhanghb97) [inline examples/BuddyLlama/CMakeLists.txt:52]: It seems we can remove this `SET_SOURCE_FILES_PROPERTIES`.

**Review by zhanghb97 — COMMENTED** [review]
I recommend that we split the container modification into another PR.

**Pull request closed without merging**